third_party/terraform/tests/resource_bigquery_table_test.go

```diff
@@ -416,6 +416,30 @@ func TestAccBigQueryDataTable_sheet(t *testing.T) {
 	})
 }
 
+func TestAccBigQueryDataTable_bigtable(t *testing.T) {
```

Comment:
This test should only be run for the `beta` provider, right? This file may need to be renamed to end in `.go.erb` and wrap this test in if/else tags to check the version.

An example can be seen here: https://github.com/GoogleCloudPlatform/magic-modules/blob/master/third_party/terraform/tests/resource_binaryauthorization_policy_test.go.erb#L41

Reply:
That's quite a weird one actually:
You can set the format to BIGTABLE in stable (search for sourceFormat in https://bigquery.googleapis.com/discovery/v1/apis/bigquery/v2/rest )
However only 2 regions are supported for the queries https://cloud.google.com/bigquery/external-data-bigtable#supported_regions_and_zones.

From the BigQuery API point of view, I don't think it requires the beta provider.
But from BigTable it does.
Not sure of the preferred approach here.